kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/templates/kroxylicious/KroxyliciousFilterTemplates.java

```diff
@@ -68,11 +96,16 @@ private static Map<String, Object> getRecordEncryptionConfigMap(TestKmsFacade<?,
                 .convertValue(testKmsFacade.getKmsServiceConfig(), new TypeReference<>() {
                 });
 
+        String kekTemplate = "KEK_$(topicName)";
+        if (testKmsFacade instanceof KubeLowKeyVaultKmsTestKmsFacade) {
+            kekTemplate = "KEK-$(topicName)";
+        }
```

Comment:
I wonder if we should just change the default template to `KEK-`. The fact that this class is aware of `KubeLowKeyVaultKmsTestKmsFacade` is a smell.